kubectl-gadget.Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Dockerfile for the kubectl-gadget plugin.
+# Since the plugin is a statically compiled golang application, any base
+# image is valid, even scratch. Alpine is used as a tradeoff between
+# size and tools available in the image.
```

Comment:
I was expecting it to be used like this:

```bash
$ docker run --rm -it kubectl-gadget trace exec
NODE             NAMESPACE        POD              CONTAINER        PCOMM            PID    PPID   RET ARGS
...
```

In that case we could use scratch as base image.

Reply:
That's one option. Another one is to run `docker run --rm -it kubectl-gadget /bin/sh` and then execute kubectl-gadget from there, it provides lower latency as it doesn't need to create a new container instance for each kubectl-gadget invocation. Alpine is around 6MB, so I'd say it's better to provide an image with some more tools than just providing the kubectl-gadget binary.